generators/csharp/sdk/src/SdkGeneratorContext.ts

```diff
@@ -87,6 +88,10 @@ export class SdkGeneratorContext extends AbstractCsharpGeneratorContext<SdkCusto
         ];
     }
 
+    public getPublicCoreAsIsFiles(): string[] {
+        return [AsIsFiles.GrpcRequestOptions];
```

Comment:
should this also be conditional on `context.hasGrpcEndpoints()`?

Reply:
Yes definitely, great catch 👍